### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by TheTechRobo**
- change "What?" to "What is it?" in readme
- change "MacOS" to "macOS" in readme
- change "worldometer" to "Worldometer" in readme
- ~~make the "about me" a full sentence in readme~~
- change "per million" to "cases per million"
- change "test" to "tests"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ Why not? 🤷
 
 ## About me
 
-[Duarte O.Carmo](https://duarteocarmo.com)
+I am [Duarte O.Carmo](https://duarteocarmo.com).
```
Comment (duarteocarmo) [inline README.md]: No really a fan of this change. Can you revert?

Comment (duarteocarmo) [reply README.md]: After that, will merge

**Review by duarteocarmo — COMMENTED** [review]
Just a couple of comments. Thanks!

File: README.md (hunk 1 above)
Comment (TheTechRobo) [reply README.md]: ok

Comment (TheTechRobo) [reply README.md]: All done

**Comment by TheTechRobo** [comment]
All done :)

**Review by duarteocarmo — APPROVED** [review]
Thanks for your contribution!

Will add changes to next release :)

**Pull request merged**